packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -107,10 +107,11 @@ class Checkbox extends StatefulWidget {
   /// design [Checkbox].
   ///
   /// If a [CupertinoCheckbox] is created, the following parameters are ignored:
-  /// [mouseCursor], [fillColor], [hoverColor], [overlayColor], [splashRadius],
-  /// [materialTapTargetSize], [visualDensity], [isError]. However, [shape] and
-  /// [side] will still affect the [CupertinoCheckbox] and should be handled if
-  /// native fidelity is important.
+  /// [hoverColor], [overlayColor], [splashRadius], [materialTapTargetSize],
+  /// [visualDensity]. [hoverColor], [overlayColor] and [splashRadius] are
+  /// ignored because a Cupertino-design checkbox does not have a radial reaction
+  /// overlay. [materialTapTargetSize] and [visualDensity] are ignored because
+  /// both are Material-design-specific configurations.
```

Comment:
Nit: I would reword this a bit. "If a [CupertinoCheckbox] is created, the following parameters are ignored because a Cupertino-design checkbox does not have a radial reaction overlay: **parameters here**. The following are ignored because both are Material-design-specific configurations: **parameters here**.`